### DISCUSSION TITLE
Использование атрибутов "стадирование" и "подтверждение" ресурса Condition.

### DISCUSSION THREAD

**Discussion by Vrodvi** [fhir-ru/core#96] [Клиническая РГ]
Карточка для обсуждения атрибутов "стадирование" и "подтверждение" для ресурса Condition.

![image](https://user-images.githubusercontent.com/92928137/159311031-2f52fcb0-6ca5-4021-964e-d2bbf77e9e2a.png)

Необходимо перевести описание этих атрибутов на русский, а также уточнить их область применения.

Некоторые проблемы с данными атрибутами:

- (1) stage  может ссылаться на ClinicalImpression или DiagnosticReport или Observation - в какой ситуации какой вариант использовать? 
Например, для стадирования по TNM - атрибут `stage` должен сослаться на отдельные Observation, описывающие T, N и M?
Или на один дополнительный Observation со стадией, у которого T, N и M - компоненты?
Или на DiagnosticReport, в котором будут указаны Observation со стадией и компонентами T, N и M?
Или стадия должна быть оформлена в виде  ClinicalImpression?

- (2) относится ли evidence ко всему состоянию (condition) или может относиться к отдельным его частям (например к стадии)?
Например, если в `stage` указана стадия "IIa", то можно ли в качестве `evidence` указать отдельные Observation, описывающие T, N и M или так делать неправильно, поскольку значения T, N и M подтверждают стадию, но не само состояние?

---
Полное описание данных атрибутов из официальной спецификации FHIR:

* stage 0..* BackboneElement 
	"Stage/grade, usually assessed formally 
	|**Оценка стадии/градации заболевания (обычно в формализованном виде)**" 
	"Clinical stage or grade of a condition. May include formal severity assessments.
	|**Клиническая стадия или градация заболевания. Элемент может ссылаться на формализованные высказывания.**"

	* stage.summary 0..1 CodeableConcept 
		"Simple summary (disease specific)
		|**Сводка в простом текстовом виде (учитывающая специфику конкретного заболевания)**" 
		"A simple summary of the stage such as \"Stage 3\". The determination of the stage is disease-specific.
		|**Простое текстовое описание стадии, например в виде \"Стадия III\". Конкретный формат написания значения стадии зависит от типа заболевания.**"
		from http://hl7.org/fhir/ValueSet/condition-stage (example)
		^binding.extension.url = "http://hl7.org/fhir/StructureDefinition/elementdefinition-bindingName"
		^binding.extension.valueString = "ConditionStage"
		^binding.description = "Codes describing condition stages (e.g. Cancer stages).
		                       |**Коды для описания стадии заболевания (например, стадирование опухолвевого процесса).**"

	* stage.assessment 0..* 
	Reference(http://hl7.org/fhir/StructureDefinition/ClinicalImpression or http://hl7.org/fhir/StructureDefinition/DiagnosticReport or http://hl7.org/fhir/StructureDefinition/Observation) 
		"Formal record of assessment
		|**Формализованное представление оценки стадии**" 	
		"Reference to a formal record of the evidence on which the staging assessment is based.
		|**Ссылка на формализованную запись о сведениях, на основании которых была оценена стадия.**"

	* stage.type 0..1 CodeableConcept 
		"Kind of staging
		|**Тип стадирование**" 
		"The kind of staging, such as pathological or clinical staging.
		|**Тип стадирования, например, клиническое (cTNM) или патологическое (pTNM) стадирование опухолевого процесса.**"
		from http://hl7.org/fhir/ValueSet/condition-stage-type (example)
		^binding.extension.url = "http://hl7.org/fhir/StructureDefinition/elementdefinition-bindingName"
		^binding.extension.valueString = "ConditionStageType"
		^binding.description = "Codes describing the kind of condition staging (e.g. clinical or pathological).
		                       |**Коды, описывающие тип стадирования заболевания (например, клиническое (cTNM) или патологическое (pTNM))**"

* evidence 0..* BackboneElement 
	"Supporting evidence
	|**Подтверждающее доказательство**" 
	"Supporting evidence / manifestations that are the basis of the Condition's verification status, such as evidence that confirmed or refuted the condition.
	|**Подтверждающие доказательства/проявления, на основании которых был установлен статус подтверждения заболевания, например, такие сведения, на основании которых заболевание было подтверждено или отклонено.**"
	^comment = "The evidence may be a simple list of coded symptoms/manifestations, or references to observations or formal assessments, or both.
	           |**Доказательством может быть простой список закодированных симптомов/проявлений, или ссылки на наблюдения и/или иные формализованные оценки.**"

	* evidence.code 0..* SU CodeableConcept 
		"Manifestation/symptom
		|**Проявление/симптом**" 
		"A manifestation or symptom that led to the recording of this condition.
		|**Проявление или симптом, который привел к появлению записи о данном состоянии.**"
		from http://hl7.org/fhir/ValueSet/manifestation-or-symptom (example)
		^binding.extension.url = "http://hl7.org/fhir/StructureDefinition/elementdefinition-bindingName"
		^binding.extension.valueString = "ManifestationOrSymptom"
		^binding.description = "Codes that describe the manifestation or symptoms of a condition.
		                       |**Коды, описывающие проявление или симптомы состояния.**"

	* evidence.detail 0..* SU Reference(http://hl7.org/fhir/StructureDefinition/Resource) 
		"Supporting information found elsewhere
		|**Внешняя подтверждающая информация**" 
		"Links to other relevant information, including pathology reports.
		|**Ссылки на сопутствующую важную информацию, включая протоколы патологоанатомических исследований.**"

**Comment by Vrodvi** [comment]
@JustAsWalker Илья, у тебя кажется ещё были замечания по поводу данных атрибутов - впиши их, пожалуйста, в данное обсуждение, если я их упустил.

**Reply by JustAsWalker** [reply]
### Примеры использования в онкологии
Что удалось найти в плане использования атрибутов **stage** и **evidence** в онкологии у коллег из других стран

**[mcode](https://hl7.org/fhir/us/mcode/2021May/index.html)**
![image](https://user-images.githubusercontent.com/99478093/159337212-69640884-60ab-4283-8586-74f32d48e71f.png)
Для [стадирования по TNM](https://hl7.org/fhir/us/mcode/2021May/StructureDefinition-mcode-cancer-stage-group.html) в mcode есть отдельные профили ресурсов `Observation`. На стадию дается ссылка в `Condition.stage`, а вот уже из стадии через `Observation.hasMember` идут ссылки на отдельные `Observation` для T, N и M.
Для [отличного от TNM стадирования](https://hl7.org/fhir/us/mcode/2021May/group-disease.html#non-tnm-staging-information) предлается в `Observation.stage` давать ссылку на `Observation` стадии, а уже из нее через `Observation.derivedFrom` - на прогностические факторы.
При этом для кода по ICD-O mcode сделал [отдельное расширение](https://hl7.org/fhir/us/mcode/2021May/StructureDefinition-mcode-histology-morphology-behavior.html)

---
[Germany: Oncology on FHIR: A Data Model for Distributed Cancer Research](https://pubmed.ncbi.nlm.nih.gov/34042895/)
![image](https://user-images.githubusercontent.com/99478093/159340120-a80f5f95-fcca-4812-8876-1f9b2cdad96e.png)
![image](https://user-images.githubusercontent.com/99478093/159340374-68acc90d-8591-4ed9-a08f-cc02228af9e3.png)
В статье немецких коллег топографию МКБ-О предлагается указывать непосредственно в ресурсе `Condition`, TNM - аналогично mcode, а вот морфологический код по ICD-O уже через `evidence`.

---
[Spain: CASIDE: A data model for interoperable cancer survivorship information based on FHIR](https://pubmed.ncbi.nlm.nih.gov/34781009/)
![image](https://user-images.githubusercontent.com/99478093/159341192-ef156e47-0633-4978-a4bb-4f6d91fb7eac.png)

В данной работе вообще все составляющие (симптомы, TNM, гистологию) предлагается указывать через `evidence`, а данные отдельно по T, N, M осям - компонентами общей стадии.

**Reply by JustAsWalker** [reply]
На мой взгляд, в работах коллег прослеживается, что чёткого понимания, что куда относить, нет. 
Мне нравится идея использования для стадии отдельных связанных между собой ресурсов `Observation`. Не совсем понимаю примеров, когда в `stage` стоит ссылаться на `DiagnosticReport` или `ClinicalImpression`. Причем для связи значений предлагаю все-таки использовать `hasMember` (если стадия действительно составная) или `derivedFrom` (чтобы указать значение прогностического фактора, по которому была поставлена такая стадия).

Что касается атрибута `evidence` - я это вижу структурированным заполнением элементов, из которых потом сложится текст "Диагноз был поставлен на основании ...". То есть `evidence` как бы должен идти последним атрибутом ресурса (собственно, он и описан в стандарте предпоследним, дальше только note), в котором объясняется, на основании чего было выбрано всё, что указано выше (код, стадия, сторона поражения и проч.). И вот здесь я уже ожидаю увидеть ссылки на протоколы инструментальных, гистологических и лабораторных исследований (`DiagnosticReport`), заключение врачей-специалистов (`ClinicalImpression`). Ресурс `Observation` в данном случае я бы предложил использовать исключительно для жалоб пациента.

Кроме того, в онкологии для подтверждения диагноза у нас еще есть тип - [по НСИ](https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.99.2.128/version/2.1) и по самому справочнику ICD-O:
![image](https://user-images.githubusercontent.com/99478093/159345793-77cab5c9-b277-499c-bf46-2424205e2750.png)
Его я вижу как группирующий `evidence.code` массива исследований или наблюдений в `evidence.detail`

---
Отвечая на вопросы из описания:
* _stage может ссылаться на ClinicalImpression или DiagnosticReport или Observation - в какой ситуации какой вариант использовать?_ - пока не совсем понимаю примеров, когда в `stage` стоит ссылаться на `DiagnosticReport` или `ClinicalImpression`
* _отдельные Observation или компоненты_ - предлагаю не использовать компоненты и выстраивать связи между `Observation` (другой вопрос - что делать если есть TNM, но нет итоговой стадии?)
* _относится ли evidence ко всему состоянию (condition) или может относиться к отдельным его частям (например к стадии)_ - как ко всей Нозологической единице в целом, так и к любой её части
* _если в stage указана стадия "IIa", то можно ли в качестве evidence указать отдельные Observation, описывающие T, N и M или так делать неправильно, поскольку значения T, N и M подтверждают стадию, но не само состояние_ - предлагаю не дублировать ссылки и составляющие TNM указывать только по связям в `stage`; пример - на МРТ нашли метастазы - M1 будет указан в `Observation` через цепочку в `stage`, а сам протокол МРТ в виде `DiagnosticReport` пойдет в `evidence`
---
Если коллеги предоставят примеры стадирования и обоснования диагноза из других областей, предлагаю обсудить уже в терминах ресурсов, как эти данные можно было бы уложить и сослаться на них через атрибуты stage / evidence

**Reply by Vrodvi** [reply]
> пока не совсем понимаю примеров, когда в stage стоит ссылаться на DiagnosticReport или ClinicalImpression

Вот возможные конфигурации для описания стадировования TNM:

1) Condition.stage ссылается на один из не связанных Observation:
```
Condition.stage: Reference(Observation: стадия IIc)
Observation: T1a
Observation: N2b
Observation: M0
Observation: стадия IIc
```
2) Condition.stage ссылается на один из связанных Observation:
```
Condition.stage: Reference(Observation: стадия IIc)
Observation: стадия IIc
- hasMember: Reference(Observation: T1a)
- hasMember: Reference(Observation: N2b)
- hasMember: Reference(Observation: M0)
```
3) Condition.stage ссылается на "интерпретацию" (ClinicalImpression) совокупности значений T, N и M как стадии:
```
Condition.stage: Reference(ClinicalImpression)
ClinicalImpression
- investigation.item: Reference(Observation: T1a)
- investigation.item: Reference(Observation: N2b)
- investigation.item: Reference(Observation: M0)
- finding.itemReference: Reference(Observation: стадия IIc)
```
4) Condition.stage ссылается на "протокол исследования" (DiagnosticReport), в котором определены значения T, N, M и стадии:
```
Condition.stage: Reference(DiagnosticReport)
DiagnosticReport
- result: Reference(Observation: стадия IIc)
- result: Reference(Observation: T1a)
- result: Reference(Observation: N2b)
- result: Reference(Observation: M0)
```
--- 

`1.` и `2.` - принципиально не отличаются, ссылка идёт на один конкретный Observation со значением стадии
`3.` ClinicalImpression - слишком "сырой" ресурс (Maturity Level: 0), лучше не использовать
`4.` DiagnosticReport - слишком "общий" ресурс, чтобы его использовать как определение "стадии"
> A diagnostic report is the set of information ... a mix of atomic results, text reports, images, and codes.

**Comment by Vrodvi** [comment]
> отдельные Observation или компоненты - предлагаю не использовать компоненты и выстраивать связи между Observation 
- согласен, этот вариант предпочтительнее (т.к. значения T, N, M имеют собственную информационную ценность и желательно их иметь в виде отдельных ресурсов)
> (другой вопрос - что делать если есть TNM, но нет итоговой стадии?)
- в этом случае предлагаю считать что стадирование отсутствует и атрибут `Condition.stage` не заполнять

**Comment by Vrodvi** [comment]
Добавил в описание перевод текстовых частей атрибутов stage и evidence.
Нужно подумать как эти текстовки дополнить, чтобы исключить неоднозначность их интерпретации.
